src/components/FavoritesBtn.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+import api from '../service/api';
+
+import '../styles/FavoritesBtn.scss';
+
+type FavoritesBtnProps = {
+  movieId: number;
+};
+
+const FavoritesBtn: React.FC<FavoritesBtnProps> = ({ movieId }) => {
+  const [isFavorite, setisFavorite] = useState(api.isIdInFavorites(movieId));
+
+  const heartUnfilledIcon = '/images/favoriteBtn/heartFilled.svg';
+  const heartFilledIcon = '/images/favoriteBtn/heartUnfilled.svg';
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>): void => {
```

Comment:
Please use more specific name for the function

Reply:
In the beginning, I believe I wanted to create a more generic name, but now it actually makes more sense that the name should be more specific. Is handleFavoritesIconClick appropriate?